grails-app/assets/javascripts/knockout-custom-bindings.js

```diff
@@ -533,19 +533,19 @@ ko.bindingHandlers.autocomplete = {
                             source: item
                         }
                     });
-                    items = [{
+                    items = items.push({
```

Comment:
Is this correct? My understanding is items we be reassigned to be the length of the new array after pushing that new value?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/push

Reply:
Thanks!